### DISCUSSION TITLE
integrazione tasmota

### DISCUSSION THREAD

**Discussion by Franco2255** [indomus/forum#2012] [GENERALE - Discussioni generiche]
Buongiorno, dopo aggiornamento alla a HOME ASSISTANT 2022.10.5 SUPERVISOR 2022.10 l'integrazione tasmota non mi trova i dispositivi MQTT
Cosa può essere successo?

**Comment by snakuzzo** [comment]
Assicurati che il `SetOption19` dei device Tasmota sia impostato a `0`.
Assicurati che venga pubblicato il topic di discovery `tasmota/discovery/...` (che è quello di default) sulla coda del broker MQTT.
Nel frattempo, controlla se ci sono anche errori e/o warning nei log

**Reply by Franco2255** [reply]
trovato avevo un problema sulla porta 1883 e non si avviava il servizio mqtt
grazie comunque
